controllers/storageoscluster/csi_operand.go

```diff
@@ -106,6 +107,74 @@ func (c *CSIOperand) Delete(ctx context.Context, obj client.Object) (eventv1.Rec
 	return nil, b.Delete(ctx)
 }
 
+// This collection depends on the order of containers at CSI deployment manifest,
+// on case of update please update the manifest too.
+var csiContainerPatches = map[string]func(c *storageoscomv1.StorageOSCluster) []string{
+	"provisioner": func(c *storageoscomv1.StorageOSCluster) []string {
+		const (
+			defaultWorkerThreads uint8 = 32
+			defaultTimeout             = 60 * time.Second
```

Comment:
Can we make the provisioner timeout 30s, we did some testing and that works best